R/pkg/tests/fulltests/test_mllib_regression.R

```diff
@@ -102,10 +102,18 @@ test_that("spark.glm and predict", {
 })
 
 test_that("spark.glm summary", {
+  # prepare dataset
```

Comment:
Can you elaborate it more? I'm not sure I get your idea here.

Reply:
You are actually reproducing the whole iris data set here, and reproducing it again in many examples. If the goal is just to change the type of one of the cols, just coerce the type of the col in the dataframe?

Reply:
Here I handcrafted tiny dataset (8 rows) to replace iris dataset (150 rows). Because iris dataset contains a column which has frequently equal string values (50 times for 3 strings) and we can't config R's string indexer, it makes trouble to produce same results for this test.

There are few discussions https://github.com/apache/spark/pull/20146#issuecomment-355764569 for this.

Reply:
I see, that's fine then. If the only issue is the type of a column in `iris` then there's probably an easier way to do this, but, doesn't matter much.